CONTRIBUTING.md

```diff
@@ -65,44 +65,59 @@ or by running `pytest` inside activated environment.
 
 
 ## Integration tests
-
 Every integration test should have `integration` mark. By default, integration tests are not execute when running `pytest`.
 
-To execute only integration tests run `hatch run integration:test` or `pytest -m integration` inside environment.
-### Connection parameters in `config.toml`
-
-Add the following connection to your `config.toml`
-
-```toml
-[connections.integration]
-host = <host>
-account = <account_name>
-user = <user>
-password = <password>
+### Account setup
+The account used for integration tests must first be set up using the `ACCOUNTADMIN` role. Run
+```bash
+snow sql -c <connection name> -f tests_integration/scripts/integration_account_setup.sql
```

Comment:
I'm working on this script, adding missing queries, fixing.

Reply:
ah cool, want me to hold off on this PR?

Reply:
No, you should merge it.